test/util/TestData.scala

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package util
+
+import play.api.libs.json._
+import play.api.mvc._
+import play.api.test.FakeRequest
+import play.mvc.Http.HeaderNames._
+import play.mvc.Http.MimeTypes
+import uk.gov.hmrc.customs.managesubscription.domain.SubscriptionCompleteStatus.SubscriptionCompleteStatus
+import uk.gov.hmrc.customs.managesubscription.domain.protocol.Eori
+import uk.gov.hmrc.customs.managesubscription.domain.{HandleSubscriptionRequest, _}
+
+object TestData {
+  val formBundleId = "dummy-bundle-id-1"
+  val sapNumber = "0123456789"
```

Comment:
To me this sapNumber is OK - it's clearly "made up".  Do you agree @RashmithReddy ?